docs/core/docker/container-images.md

```diff
@@ -0,0 +1,126 @@
+---
+title: .NET container images
```

Comment:
Going to cc @richlander on this page, since his team is driving the direction of them. Having a page like this would be great for pointing other documentation towards IMO.

Reply:
Along these lines, can you add a backlink from https://learn.microsoft.com/en-us/dotnet/core/whats-new/dotnet-8#container-images?

Reply:
Done